### ISSUE TITLE
HCN and other weak acids/bases nearly impossible to extract ions from

### ISSUE THREAD

**Issue by flynnvali** [petrolpark/Destroy#547]
This was fixed in the community addition by manually adding neutralization rxns to HCN with OH ions, however in base Destroy, obtaining CN and other ions like it is nearly impossible in reasonable concentrations and amounts.

I am unsure if this should be labeled as a bug or suggestion since everything is technically working as intended, however it is presently impossible to obtain any cyanated molecules such as acrylonitrile in survival mode in any reasonable amounts in a reasonable amount of time, which is vital for survival progression.

A simple example would be performing a cyanide addition to vinyl chloride to create acrylonitrile in an HCN and water solution. It takes minutes to reach even a single mmol of acrylonitrile, requiring hours of AFKing to create a single mole of acrylonitrile for polymerization into PAN.

(Tested in Snapshot 11)

**Comment by flynnvali** [comment]
hi! poking in here because 0.1 is getting close to leaving the snapshot stage, CN- production in particular should be addressed before 0.1's release because of how much it affects progression and generally being able to do complex reactions in survival mode, such as PAN and PMMA. (a simple manual neutralization rxn with hydroxide ala boric acid like in the community edition would probably work)

**Comment by TheRedCrafter344** [comment]
The actual problem here is that the integration scheme destroy uses is completely inadequate. Adding a reaction HCN + OH <--> CN + H2O would probably work, but it would not fix the underlying issue.

**Comment by flynnvali** [comment]
Precisely, but iirc the entire chemistry system is being rewritten so a simple neutralization rxn would be a good band-aid fix in the meantime to allow progression

**Comment by petrolpark** [comment]
Patchwork fix was included in snapshot 15.
